src/components/ForecastChart.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Line } from 'react-chartjs-2';
+import PropTypes from 'prop-types';
+
+import { getForecastData } from '../reducers';
+
+import './ForecastChart.css';
+
+class ForecastChart extends React.Component {
+  render() {
+    const { forecastData } = this.props;
+    const label = "Max temperature";
+    const backgroundColor = "rgba(174,234,0,0.4)";
+    const borderColor = "rgba(174,234,0,1)"
+    const dates = forecastData.map(forecast => forecast.date);
+    const temperatures = forecastData.map(forecast => forecast.temp);
+
+    const data = {
```

Comment:
Why you creating data instead of passing data to `Line` and using them there they suppose to be used? This is just a question

Reply:
You mean doing it in the component directly?

like

```
<Line 
  data={
    labels: dates,
    datasets: []
  } 
/>
```

I thought that extracting it somewhere nearer the top would make it slightly more readable, but with such a simple component this may be redundant, you're right.